webapp/src/main/java/com/box/l10n/mojito/rest/scheduledjob/ScheduledJobWS.java

```diff
@@ -0,0 +1,169 @@
+package com.box.l10n.mojito.rest.scheduledjob;
+
+import com.box.l10n.mojito.entity.ScheduledJob;
+import com.box.l10n.mojito.service.scheduledjob.ScheduledJobDTO;
+import com.box.l10n.mojito.service.scheduledjob.ScheduledJobManager;
+import com.box.l10n.mojito.service.scheduledjob.ScheduledJobRepository;
+import com.box.l10n.mojito.service.scheduledjob.ScheduledJobResponse;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobKey;
+import org.quartz.SchedulerException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+
+/**
+ * Webservice for viewing, enabling/disabling and triggering scheduled jobs.
+ *
+ * @author mattwilshire
+ */
+@RestController
+@ConditionalOnProperty(value = "l10n.scheduledJobs.enabled", havingValue = "true")
+public class ScheduledJobWS {
+  static Logger logger = LoggerFactory.getLogger(ScheduledJobWS.class);
+  private final ScheduledJobRepository scheduledJobRepository;
+  private final ScheduledJobManager scheduledJobManager;
+
+  @Autowired
+  public ScheduledJobWS(
+      ScheduledJobRepository scheduledJobRepository, ScheduledJobManager scheduledJobManager) {
+    this.scheduledJobRepository = scheduledJobRepository;
+    this.scheduledJobManager = scheduledJobManager;
+  }
+
+  private final ResponseEntity<ScheduledJobResponse> notFoundResponse =
+      createResponse(
+          HttpStatus.NOT_FOUND, ScheduledJobResponse.Status.FAILURE, "Job doesn't exist");
+
+  @RequestMapping(method = RequestMethod.GET, value = "/api/jobs")
+  @ResponseStatus(HttpStatus.OK)
+  public List<ScheduledJobDTO> getAllJobs() {
+    List<ScheduledJob> scheduledJobs = scheduledJobRepository.findAll();
+    return scheduledJobs.stream().map(ScheduledJobDTO::new).collect(Collectors.toList());
+  }
+
+  @RequestMapping(method = RequestMethod.GET, value = "/api/jobs/{id}")
+  @ResponseStatus(HttpStatus.OK)
+  public ScheduledJobDTO getJob(@PathVariable UUID id) {
+    return scheduledJobRepository
+        .findByUuid(id.toString())
+        .map(ScheduledJobDTO::new)
+        .orElseThrow(
+            () ->
+                new ResponseStatusException(HttpStatus.NOT_FOUND, "Job not found with id: " + id));
+  }
+
+  @RequestMapping(method = RequestMethod.POST, value = "/api/jobs/{id}")
```

Comment:
Just looking at a POST to "/api/jobs/{id}" suggests to me that we might be updating some information for that job or submitting a new/updated job definition (as the get at on the same path/route returns us back the information about the job).
(We might also want to reserve the POST/PUT to that route for dynamically creating job definitions in the future)

Thinking of using something more explicit here, similar to enable/disable, e.g.:
"/api/jobs/{id}/trigger" ?

Reply:
Done @garionpin